Description: Fix borders in pstoimg
 This patch fixes the borders of images in pstoimg.
 The origin of this patch is actually unknown. Feel free to tell me and I'll
 include it.
Author: Roland Stigge <stigge@antcom.de>

--- latex2html-2008-debian1.orig/pstoimg.pin
+++ latex2html-2008-debian1/pstoimg.pin
@@ -1289,9 +1289,9 @@ sub crop_scale_etc {
     my $edge = $1;
     my $croparg = '';
     if($edge =~ /b/i) {
-      $croparg = "-bot $PNMCROPOPT ";
+      $croparg = "-bot -sides $PNMCROPOPT ";
     } elsif($edge =~ /[tlr]/i) {
-      $croparg = "-$edge $PNMCROPOPT ";
+      $croparg = "-$edge -sides $PNMCROPOPT ";
     } elsif($edge =~ /s/i) {
       #RRM: shave at most 1-2 rows of white from the bottom
 #if @pipes@

